zipkin-server/src/main/java/zipkin2/server/internal/elasticsearch/ZipkinElasticsearchStorageProperties.java

```diff
@@ -346,6 +349,10 @@ public void setSsl(Ssl ssl) {
     this.ssl = ssl;
   }
 
+  public String getPriority() { return priority; }
+
+  public void setPriority(String priority) { this.priority = priority; }
```

Comment:
If this is a string to coerce empty to null, make a comment (also coerce empty to null :P) but still use an integer in the builder. I'm not actually sure what Spring does to an integer field set to ""